channels/kr.m3u

```diff
@@ -115,6 +115,8 @@ http://live.worktv.or.kr:1935/live/wowtvlive1.sdp/playlist.m3u8
 http://slive.ytn.co.kr/ytn/_definst_/ytn_stream_20190306/playlist.m3u8
 #EXTINF:-1 tvg-id="YTNDMB.kr" tvg-country="KR" tvg-language="Korean" tvg-logo="https://i.imgur.com/fdFQonN.jpg" group-title="",YTN DMB (270p)
 http://slive.ytn.co.kr:1935/dmb/ydlive_20140419_1/playlist.m3u8
+#EXTINF:-1 tvg-id="YTNSCIENCE.kr" tvg-country="KR" tvg-language="Korean" tvg-logo="https://i.imgur.com/FIzME5L.jpg" group-title="Science",YTN SCIENCE 
+https://iptv.dc-team.com/query?url=https://www.youtube.com/channel/UCZdBJIbJz0P9xyFipgOj1fA/live
```

Comment:
Source online
![LIVE  대한민국 과학 채널 YTN사이언스📺 13-0-1 screenshot](https://user-images.githubusercontent.com/7253922/135714992-7376214c-7c4f-44d8-b446-937c3f3eff6b.png)

but the link does not work
![image](https://user-images.githubusercontent.com/7253922/135715002-4bb77b63-1ce1-4454-a2a9-4017f7afc04f.png)

Reply:
This link is ok (query link always return 302 so check links is wrong)

Reply:
well it work for me https://i.imgur.com/VaQZ8EZ.png could it be geo ?

Reply:
Works on my end, Southwest, US

Reply:
Works here, Romania

![image](https://user-images.githubusercontent.com/30985701/135719414-4a80c979-2604-4e89-a8bc-33b65f75e914.png)

Reply:
now it works